npm/private/yaml.bzl

```diff
@@ -56,11 +56,9 @@ def parse(yaml):
             fail(msg)
 
         if starlark["error"] != None:
-            # buildifier: disable=print
-            print("YAML parse error: ", starlark["error"])
-            return None
+            return None, starlark["error"]
```

Comment:
nit: update return info in the docstring above

Reply:
You mean prepend the `YAML parse error` string?

Reply:
IMO we should leave that off. When `yaml.parse` returns an error I don't think it's worth prefixing it...? 🤷

Reply:
No, I mean in the docstring for this starlark function mention that the return code is now a tuple

Reply:
Currently it states,
```
    Returns:
        An equivalent mapping to native starlark types
    """
```
which is now out of date